.github/workflows/deploy_api.yml

```diff
@@ -0,0 +1,75 @@
+# To debug, it's recommended you modify / use the version in the test-actions repo:
+# https://github.com/covid-projections/test-actions/blob/master/.github/workflows/deploy_api.yml
+
+name: Build / Publish API artifacts to data.covidactnow.org
+
+on:
+  # JHU posts new data at 11:30 (UTC).
+  # covid-data-public fetches it at 12:00 (UTC).
+  # So we rebuild / publish the API at 12:30 (UTC).
+  schedule:
+   - cron: '30 12 * * *'
```

Comment:
Pretty sure this is right, but I don't usually write cron jobs, so please double check me.  The intention is to run it at 12:30 UTC (30 minutes after noon) daily.

Reply:
looks good!